app/views/receipt_mailer/receipt_email.html.erb

```diff
@@ -6,10 +6,10 @@ your <%= @thesis.graduation_month %> <%=
 @thesis.graduation_year %> thesis "<%= @thesis.title %>" to the MIT Libraries.
 </p>
 
-<p>If you submitted a May 2020 theses, please note that only the ProQuest form,
-title page, and abstract should have been submitted here. The actual thesis
-should go directly to your department. For degree periods prior to May 2020, we
-accept all electronic copies via this form.</p>
+<p>If you submitted a May or September 2020 thesis, please note that only the
+ProQuest form, title page, and abstract should have been submitted here. The
+actual thesis should go directly to your department. For degree periods prior to
+May 2020, we accept all electronic copies via this form.</p>
```

Comment:
This seems to refer to the starting point for the new process, so I've left it unchanged